### ISSUE TITLE
Use Date (or DateTime) in `extent`

### ISSUE THREAD

**Issue by Moelf** [queryverse/VegaLite.jl#286]
I noticed that at the moment you need to convert a `DateTime` object by `(1e3 * datetime2unix(date))` to get it working in the `extent` keyword of anything, can we fix it so that we can just pass DateTime[] to it?

**Comment by davidanthoff** [comment]
Yes, we should fix that! Did you find some vega-lite documentation that describes how dates and extent should be combined?

**Comment by Moelf** [comment]
I don't think so, the documentation regarding `extent` (here: https://vega.github.io/vega-lite/docs/regression.html) just says `Number`, which doesn't for a temporal axis, probably means millisecond...

**Comment by davidanthoff** [comment]
Hm, we should probably first figure out what the vega-lite story for `extent` and date time stuff is. It almost seems to me that they should just add native support for date time in the `extent` field, right?
